pkg/metadata/inventories/README.md

```diff
@@ -61,7 +61,7 @@ The payload is a JSON dict with the following fields
     enabled, otherwise the field is omitted.
   - `feature_cws_enabled` - **bool**: True if the Cloud Workload Security is enabled (see: `runtime_security_config.enabled`
     config option).
-  - `feature_process_enabled` - **bool**: True if the Process Agent is enabled (see: `process_config.enabled` config
+  - `feature_process_enabled` - **bool**: True if the Process Agent is enabled (see: `process_config.process_collection.enabled` config
```

Comment:
Just for my understanding, are we currently set it to `false` if only container check is enabled?

Reply:
Yes, the code that this documentation corresponds with is [here](https://github.com/DataDog/datadog-agent/pull/10555/files#diff-86c59ed4a12c5caf4344660a96189233bb637d93b612ecc0922b1237a6f6e401R283). In this case we set this value to false if `process_config.enabled` is falsy, which means that the process check is disabled.